app/lib/server/server.go

```diff
@@ -180,7 +189,8 @@ func InteractiveConfig(binaryName string) {
 		log.Fatal("Impossible to parse public key:", err)
 	}
 
-	server := config.NewServerToml(network.Suite, public, publicAddress)
+	log.Print(publicAddress)
+	server := config.NewServerToml(network.Suite, public, publicAddress, cert)
```

Comment:
Yes, every new conode will use TLS. I could add a switch `-notls` to `setup` which would make a PlainTCP connection.